data-sources/citizens/generate-citizen-count-csv.ts

```diff
@@ -1,3 +1,5 @@
+export {}
```

Comment:
@johnmark13 Not sure, but is it best-practice to put this line at the _bottom_ of the file instead of at the top?  At least, that's what I did when I had a similar (?) problem with the Cypress tests: https://github.com/nation3/app/blob/main/ui/cypress/e2e/app.cy.ts

Reply:
No worries, I'll move them 👍